.github/workflows/run_tests.yml

```diff
@@ -15,6 +17,7 @@ jobs:
       matrix:
         version:
           - '1.4'
+          - '1.5'
```

Comment:
This won't test on the 1.5 prereleases as I understand it — looking at the logs I think this just ran the tests on 1.4.2 twice.  I _think_ you can do `'^1.5-'`, which will grab the latest version of 1.5, including prereleases.

Reply:
Oh, I will investigate this. Thanks for the pointer.